### ISSUE TITLE
clearing pending jobs

### ISSUE THREAD

**Issue by ahmedalmulki** [Dionera/laravel-beanstalkd-admin-ui#17]
can I flush all pending jobs like a flush queue or clear queue something like that, if I have 20000 jobs ready in admin panel and I want to delete them that because I made some mistake, is that possible?

**Comment by ksassnowski** [comment]
I think this is similar to #5, isn't it? I haven't gotten around to implementing something like that but I'd happily accept a merge request for it. I haven't looked into it but I can't imagine that it would be particularly hard to implement.

**Comment by ahmedalmulki** [comment]
Yes, it is similar to this section"Is there a way to remove all jobs from a tube? We have a need to empty them sometimes", in Larvel community suggest to use a package like [this](https://github.com/morrislaptop/laravel-queue-clear/blob/master/src/Console/QueueClearCommand.php), but it will be better if this package support that to clear all queue from Beanstalkd UI admin panel.

**Comment by ksassnowski** [comment]
It looks like Beanstalkd itself doesn't expose a command to flush a tube. See [this open](https://github.com/beanstalkd/beanstalkd/issues/25) issue from 2009 on the beanstalkd repository.

That means we would have to build the command manually by looping over all commands in the tube. This shouldn't be too hard to implement and I would appreciate a pull request for it. Otherwise I don't have an ETA on when I will implement this.
